recipes/odo/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "odo" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "e892ffe58e87d6b1c12838c4ab252bd0ae8c0ec43e5179409309ded4929905d6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
I tried that, but the build fails without these. (Note that I didn't add numpy in the build section)